src/Extensions/azure-storage/azure-storage/Microsoft.Diagnostics.Monitoring.AzureStorage.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>$(ToolTargetFrameworks)</TargetFrameworks>
+		<RuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</RuntimeIdentifiers>
+		<PackAsToolShimRuntimeIdentifiers>linux-x64;linux-musl-x64;win-x64</PackAsToolShimRuntimeIdentifiers>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<RootNamespace>Microsoft.Diagnostics.Monitoring.AzureStorage</RootNamespace>
+		<ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I would recommend against enabling these. Since you will have to backport to `release/6.x`, I don't think this is going to work for the `netcoreapp3.1` TFM.